flags/14.0/vtctldclient.txt

```diff
@@ -80,134 +84,42 @@ Available Commands:
-      --xtrabackup_backup_flags string                                   flags to pass to backup command. These should be space separated and will be added to the end of the command
-      --xtrabackup_root_path string                                      directory location of the xtrabackup and xbstream executables, e.g., /usr/bin
-      --xtrabackup_stripe_block_size uint                                Size in bytes of each block that gets sent to a given stripe before rotating to the next stripe (default 102400)
-      --xtrabackup_user string                                           User that xtrabackup will use to connect to the database server. This user must have all necessary privileges. For details, please refer to xtrabackup documentation.
+      --action_timeout duration                timeout for the total command (default 1h0m0s)
+      --alsologtostderr                        log to standard error as well as files
+      --grpc_auth_static_client_creds string   When using grpc_static_auth in the server, this file provides the credentials to use to authenticate with server.
+      --grpc_compression string                Which protocol to use for compressing gRPC. Default: nothing. Supported: snappy
+      --grpc_enable_tracing                    Enable gRPC tracing.
+      --grpc_initial_conn_window_size int      gRPC initial connection window size
+      --grpc_initial_window_size int           gRPC initial window size
+      --grpc_keepalive_time duration           After a duration of this time, if the client doesn't see any activity, it pings the server to see if the transport is still alive. (default 10s)
+      --grpc_keepalive_timeout duration        After having pinged for keepalive check, the client waits for a duration of Timeout and if no activity is seen even after that the connection is closed. (default 10s)
+      --grpc_max_message_size int              Maximum allowed RPC message size. Larger messages will be rejected by gRPC with the error 'exceeding the max size'. (default 16777216)
+      --grpc_prometheus                        Enable gRPC monitoring with Prometheus.
+  -h, --help                                   help for vtctldclient
+      --keep_logs duration                     keep logs for this long (using ctime) (zero to keep forever)
+      --keep_logs_by_mtime duration            keep logs for this long (using mtime) (zero to keep forever)
+      --log_backtrace_at traceLocation         when logging hits line file:N, emit a stack trace (default :0)
+      --log_dir string                         If non-empty, write log files in this directory
+      --log_rotate_max_size uint               size in bytes at which logs are rotated (glog.MaxSize) (default 1887436800)
+      --logtostderr                            log to standard error instead of files
+      --mysql_server_version string            MySQL server version to advertise.
```

Comment:
how did this get into vtctldclient??!!!

Reply:
`local` client => `ApplyVSchema` => `sqlparser.Parse` => depends on `mysql_server_version`

Reply:
(among other commands that parse sql)